content/docs/5.1/reference/architecture/overview.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Overview
+description: Core concepts and architecture behind TiKV.
+menu:
+    "5.1":
+        parent: Architecture
+        weight: 1
+---
+
+This page discusses the core concepts and architecture behind TiKV, including:
+
+* The [APIs](#apis) that applications can use to interact with TiKV
+* The basic [system architecture](#system) underlying TiKV
+* The anatomy of each [instance](#instance) in a TiKV installation
+* TiKV's terminology including [Placement Driver](#placement-driver), [Store](#store), [Region](#region), and [Node](#node)
+* TiKV's [transaction model](#transactions)
+* The role of the [Raft consensus algorithm](#raft) in TiKV
+* The [origins](#origins) of TiKV
+
+## APIs
+
+TiKV provides two APIs that you can use to interact with it:
+
+| API           | Description                                                                           | Atomicity     | Use when...                                                                                                            |
+|:------------- |:------------------------------------------------------------------------------------- |:------------- |:---------------------------------------------------------------------------------------------------------------------- |
+| Raw           | A lower-level key-value API for interacting directly with individual key-value pairs. | Single key    | Your application requires low latency instead of distributed transactions or multi-version concurrency control (MVCC). |
+| Transactional | A higher-level key-value API that provides ACID semantics                             | Multiple keys | Your application requires distributed transactions and/or MVCC.                                                        |
+
+## System architecture {#system}
+
+The overall architecture of TiKV is illustrated below:
+
+{{< figure
+    src="/img/basic-architecture.png"
+    caption="The architecture of TiKV"
+    alt="TiKV architecture diagram"
+    width="70" >}}
+
+## TiKV instance {#instance}
+
+The architecture of each TiKV instance is illustrated below:
+
+{{< figure
+    src="/img/tikv-instance.png"
+    caption="TiKV instance architecture"
+    alt="TiKV instance architecture diagram"
+    width="60" >}}
+
+
+## Placement driver (PD) {#placement-driver}
+
+The TiKV placement driver is the cluster manager of TiKV, which periodically checks replication constraints to balance load and data automatically across nodes and regions in a process called **auto-sharding**.
+
+## Store
+
+There is a [RocksDB](https://rocksdb.org) database within each **Store** and it stores data into the local disk.
+
+## Region
```

Comment:
May be better to explain peer after this.

Reply:
```
## Peer

A replica of a Region is called a peer. Multiple peers of the same Region replicate data via the Raft consensus algorithm, so peers are also members of a Raft instance.
```